tailcfg/tailcfg.go

```diff
@@ -869,6 +869,7 @@ type Hostinfo struct {
 	ShareeNode      bool           `json:",omitempty"` // indicates this node exists in netmap because it's owned by a shared-to user
 	NoLogsNoSupport bool           `json:",omitempty"` // indicates that the user has opted out of sending logs and support
 	WireIngress     bool           `json:",omitempty"` // indicates that the node wants the option to receive ingress connections
```

Comment:
oh, this one's already just a plain bool with omitempty, great.

maybe we can say that as of capver 113 (or whatever), we don't send WireIngress=true if IngressEnabled=true (which can then be a plain bool too?) because (false, true) doesn't make any sense, really